cli/src/cmd_auth.rs

```diff
@@ -226,7 +249,8 @@ impl CmdAuthLogin {
             // Do an OAuth 2.0 Device Authorization Grant dance to get a token.
             let device_auth_url =
                 DeviceAuthorizationUrl::new(format!("{}device/auth", &self.host))?;
-            let client_id = &ctx.config().client_id;
+            // TODO what's the point of this?
+            let client_id = Uuid::new_v4();
```

Comment:
I think this was initially written by @plotnick , I'm pretty sure there was a good reason

Reply:
Reconstructing from distant memories and a quick perusal of the spec... We don't currently assign clients any kind of unique identifier, so a random UUID is about the best we can do here. It's not holding any weight, but it's not optional in the OAuth protocol.

Reply:
Yeah. The question was to myself. I saw its use by the oauth dance and wanted to investigate if there's something meaningful here or if we should document it in the code as unimportant.